env/src/main/scala/tofu/env/bio/EnvBio.scala

```diff
@@ -0,0 +1,42 @@
+package tofu.env.bio
+
+import monix.eval.Task
+
+sealed trait EnvBio[-R, +E, +A] {
+  def run(ctx: R): Task[Either[E, A]]
```

Comment:
I'm not sure if we need `sealed` here
Constructing such element using just lambda to `Task[Either[E, A]]` should be nice